doc/source/tune/examples/tune-aim.ipynb

```diff
@@ -0,0 +1,198 @@
+    "(tune-aim-ref)=\n",
+    "\n",
+    "[Aim](https://aimstack.io) is an easy-to-use and supercharged open-source experiment tracker.\n",
+    "Aim logs your training runs, enables a well-designed UI to compare them, and provides an API to query them programmatically.\n",
+    "\n",
+    "```{image} /images/aim_logo_full.png\n",
+    ":align: center\n",
+    ":alt: Aim\n",
+    ":width: 100%\n",
+    ":target: https://aimstack.io\n",
+    "```\n",
+    "\n",
+    "Ray Tune currently offers built-in integration with Aim.\n",
+    "The {ref}`AimCallback <tune-aim-logger>` automatically logs metrics that are reorted to Tune by using the Aim API.\n",
+    "\n",
+    "\n",
+    "```{contents}\n",
+    ":backlinks: none\n",
+    ":local: true\n",
+    "```\n",
+    "\n",
+    "## Running An Aim Example\n",
+    "\n",
+    "The following example demonstrates how the `AimCallback` works in a sample project.\n",
+    "Begin by importing the necessary modules:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "100bcf8a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "import ray\n",
+    "from ray import air, tune\n",
+    "from ray.air import session\n",
+    "from ray.tune.logger.aim import AimCallback"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "9346c0f6",
+   "metadata": {},
+   "source": [
+    "Next, define a simple `train_function` (a Tune `Trainable`) that reports a random loss to Tune.\n",
+    "The objective function itself is not important for this example, as our main focus is on the integration with Aim."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e8b4fc4d",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def train_function(config):\n",
+    "    for _ in range(50):\n",
+    "        loss = config[\"mean\"] + config[\"sd\"] * np.random.randn()\n",
+    "        session.report({\"loss\": loss})"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "831eed42",
+   "metadata": {},
+   "source": [
+    "Here is an example of how you can use the `AimCallback` to set up a simple grid-search Tune run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "52988599",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "def tune_with_callback():\n",
+    "     \"\"\"An example of utilizing an AimCallback\"\"\"\n",
+    "     tuner = tune.Tuner(\n",
+    "         train_function,\n",
+    "         tune_config=tune.TuneConfig(\n",
+    "             metric=\"loss\",\n",
+    "             mode=\"min\",\n",
+    "         ),\n",
+    "         run_config=air.RunConfig(\n",
+    "             callbacks=[\n",
+    "                 AimCallback(repo=\"ray_tune\", experiment_name=\"ray tune example\", as_multirun=True)\n",
```

Comment:
We should provide one example where the default `AimCallback()` is passed in, then explain in another cell the other parameters that are possible. Something like this:

```python
    callbacks=[AimLoggerCallback()],
# ...
tuner.fit()
```

```markdown
In the example above, we used the default configuration for the `AimLoggerCallback`. 
There are a few options that can be configured as arguments to the callback. For example, 
setting `AimLoggerCallback(repo="/path/to/repo")` will log results to the Aim repo at that 
filepath, which could be useful if you have a central location where the results of multiple 
Tune experiments are stored. Relative paths to the working directory where the script is 
launched can be used as well. By default, the repo will be set to the experiment log 
directory. See [the API reference](tune-aim-logger) for more configurations.
```